packages/cli/test/metadata/should-autoinstall.test.ts

```diff
@@ -0,0 +1,38 @@
+import test from 'ava';
+import {shouldAutoinstall} from '../../src/metadata/handler'
+
+test('should return false when no adaptor is provided', (t) => {
+  const adaptor = '';
+  const result = shouldAutoinstall(adaptor);
+  t.is(result, false);
+});
+
+test('should return false for adaptors with "test=" prefix', (t) => {
+  const adaptor = 'test=/home/satyammattoo/kit/integration-tests/cli/modules/test';
```

Comment:
Hmm, it's a silly aesthetic thing, but we don't need such a long path here.

Can we replace this (and the next path on line 17) with something shorter and simpler, like:
```
const adaptor = 'test=/repo/modules/test';
```